{"name": "test_login[admin-]", "status": "failed", "statusDetails": {"message": "AssertionError: 失败原因：密码为空\nassert False == True", "trace": "user = 'admin', psw = ''\n\n    @pytest.mark.parametrize(\"user, psw\", test_login_data)\n    def test_login(user, psw):\n        '''登录用例'''\n        result = login(user, psw)\n>       assert result == True, \"失败原因：密码为空\"\nE       AssertionError: 失败原因：密码为空\nE       assert False == True\n\ntest_demo13.py:23: AssertionError"}, "description": "登录用例", "parameters": [{"name": "user", "value": "'admin'"}, {"name": "psw", "value": "''"}], "start": 1615054278852, "stop": 1615054278853, "uuid": "8879be05-1ee9-497e-ad3d-e24cd5b9fa3c", "historyId": "063922d3f59aefe52759b5322bb18789", "fullName": "test_demo13#test_login[admin-]", "labels": [{"name": "parentSuite"}, {"name": "suite", "value": "test_demo13"}, {"name": "subSuite"}, {"name": "host", "value": "LHL"}, {"name": "thread", "value": "22228-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_demo13"}]}